src/puppetlabs/trapperkeeper/config.clj

```diff
@@ -59,6 +59,12 @@
     #{".yaml" ".yml"}
     (yaml/parse-string (slurp file))))
 
+(defn override-restart-file-from-cli-data
+  [config-data cli-data]
+  (if-let [cli-restart-file (:restart-file cli-data)]
+    (assoc-in config-data [:global :restart-file] cli-restart-file)
```

Comment:
So it would get logged in two different places at startup (stdout and the application's configured log with logback)? That seems fine to me, especially since we don't expect it to happen very often.